source/config.js

```diff
@@ -104,6 +106,10 @@ const defaultConfig = {
   // "pluginConfigs": { "gerrit": { "disabled": true } }
   pluginConfigs: {},
 
+  // full path to git executable. If not set, system default will be used
+  // (in this case, git should be available on PATH)
+  gitCommand: null,
```

Comment:
Probably it might also be useful to set `gitSshCommand` - I'm currently on windows and it is not very convenient to set system-wide environment variables :(

Simple use-case: on windows `putty` is widely used ssh client, which is a GUI. It also has `plink` command, which is a cli command and meant to be a drop-in replacement for `ssh` on windows (supports same options as latter). And the last - the `pageant` or the putty authentication agent, which allows to add your keys and get them delivered to either `putty` or `plink` on request (nothing new here - actually they just reflect standard ssh client features).

So despite I have bash/git/ssh/etc on windows (thanks to [msys2](http://www.msys2.org/)), it's more convenient to use `plink` instead of standard `ssh` (which just doesn't work with pageant).